### PR TITLE
Viewflow

### PR DESCRIPTION
**Pull request by JoannaMusialik**

### CODE REVIEW

File: apsi_diploma/diploma_app/models.py [hunk 1]
```diff
@@ -1,3 +1,88 @@
...
+    topic_title = models.CharField(max_length=500)
+    topic_description = models.CharField(max_length=5000)
+    topic_approved = models.BooleanField(default=False)
+    dissertation_file = models.FileField()
+    dissertation_accepted = models.BooleanField(default=False)
+    paper_type = models.CharField(
+        max_length=2,
+        choices=PAPER_TYPE_CHOICES,
+    )
+    supervisor_review = models.FileField()
```
Comment (Latkos) [inline apsi_diploma/diploma_app/models.py]: Should it be a file field or rather a text field? Let's align on that, I am not too sure.

Comment (JoannaMusialik) [reply apsi_diploma/diploma_app/models.py]: I think text field will be better.

File: apsi_diploma/apsi_diploma/settings.py [hunk 2]
```diff
@@ -75,16 +80,24 @@
 
 WSGI_APPLICATION = "apsi_diploma.wsgi.application"
 
+# DATABASES = {
```
Comment (Latkos) [inline apsi_diploma/apsi_diploma/settings.py]: Just remove it

**Review by Latkos — APPROVED** [review]
All good, approving. Resolve the merge conflicts of course

**Pull request merged**